meta/store.go

```diff
@@ -1389,38 +1389,33 @@ func (s *Store) UserCount() (count int, err error) {
 	return
 }
 
-// PrecreateShardGroups creates shard groups whose endtime is before the cutoff time passed in. This
-// avoid the need for these shards to be created when data for the corresponding time range arrives.
-// Shard creation involves Raft consensus, and precreation avoids taking the hit at write-time.
-func (s *Store) PrecreateShardGroups(cutoff time.Time) error {
+// PrecreateShardGroups creates shard groups whose endtime is before the future time passed in, but
+// is still yet to expire. This is to avoid the need for these shards to be created when data for
+// the corresponding time range arrives. Shard creation involves Raft consensus, and precreation
+// avoids taking the hit at write-time.
+func (s *Store) PrecreateShardGroups(now, future time.Time) error {
```

Comment:
`now` and `future` could maybe named a little better.  Maybe `from` and `to`?  e.g. `PrecreateShardGroups(from, to time.Time)`?  Seems like the args are defining a range of time that shard groups should be created between, is that correct?